script.js

```diff
@@ -1,4 +1,122 @@
+
+// add a win loss record, output draws and winning percentage
+
+var STONE = `stone`;
+var SCISSORS = `scissors`;
+var PAPER = `paper`;
+var REVERSED_SCISSORS = `reversed scissors`;
+var REVERSED_PAPER = `reversed paper`;
+var REVERSED_STONE = `reversed stone`;
+
+var totalGamesWon = 0;
+var totalComWins = 0;
+var totalGamesPlayed = 0;
+var totalDrawGamesPlayed = 0;
+var userName = ``;
+var spsGame = `regular`
+var currentGameMode = `enter user name`
+
+var getWinRateInfo = function () {
+  var winRate = Math.floor(totalGamesWon/totalGamesPlayed *100);
+
+  if (totalGamesPlayed == 0) {
+    return `This is your first game!`
+  }
+  return `You Won ` +totalGamesWon + ` games.`
+  + `<br><br>Computer Won `+ totalComWins + ` games.`
+  +`<br></br>You drawed ${totalDrawGamesPlayed} games.`
+  +`<br><br>You win ` + winRate + `% of all games played.` 
+}
+
+// Assign a choice to computer via randomNumber function
+
+var generateChoice = function() {
+  var randomComChoice = randomNumber();
```

Comment:
I'll use this line to illustrate why it is best practice to use verbs when naming helper functions: had you done so, you wouldn't have to struggle to come up with a synonym for randomNumber (i.e. randomChoice). Instead, this line would look like this: 
`var randomNumber = getRandomNumber()` <-- Notice how they make sense semantically?